ETWAnalyzer_uTest/EventDump/DumpFileTests.cs

```diff
@@ -642,14 +653,44 @@ public void PrintIsSummaryTotalsNullFileMode()
         {
             using var testOutput = new ExceptionalPrinter(myWriter);
             var data = CreateTestData();
+
+            DumpFile fileDumper = new()
+            {
+                TopN = new SkipTakeRange(),
+                TopNProcesses = new SkipTakeRange(),
+            };
+
+            StringWriter writer = new();
+            Console.SetOut(writer);
+
+            fileDumper.PrintData(data);
+
+            testOutput.Add(writer.ToString());
+            string[] lines = writer.ToString().Split(myNewLineSplitChars, StringSplitOptions.RemoveEmptyEntries);
+            Assert.Equal(8, lines.Length);
+            Assert.Contains("Process Count: 2", lines[7]);
+            Assert.Contains("TotalTime: 0.00000 s", lines[7]);
+            Assert.Contains("File/s Total with 0 accessed file/s", lines[7]);
+        }
+
+
+        [Fact]
+        public void PrintIsSummaryTotalsSingleLineTest()
+        {
+            using var testOutput = new ExceptionalPrinter(myWriter);
+            var data = CreateTestData();
+
             DumpFile fileDumper = new()
             {
                 TopN = new SkipTakeRange(),
                 TopNProcesses = new SkipTakeRange(),
             };
+
             StringWriter writer = new();
             Console.SetOut(writer);
+
             fileDumper.PrintData(data);
+
             testOutput.Add(writer.ToString());
             string[] lines = writer.ToString().Split(myNewLineSplitChars, StringSplitOptions.RemoveEmptyEntries);
             Assert.Equal(8, lines.Length);
```

Comment:
There should be a negative assert that no substring with the string "File/s Total" is present.